Tests.Integration.Presentation.Web/Organizations/V2/OrganizationGridConfigurationAPIV2Test.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Core.DomainModel;
+using Core.DomainModel.Organization;
+using Presentation.Web.Models.API.V1;
+using Presentation.Web.Models.API.V2.Internal.Response.Organizations;
+using Tests.Integration.Presentation.Web.Tools;
+using Tests.Toolkit.Patterns;
+using Xunit;
+
+namespace Tests.Integration.Presentation.Web.Organizations.V2
+{
+    public class OrganizationGridConfigurationApiV2Test : WithAutoFixture
+    {
+
+        [Fact]
+        public async Task LocalAdminCanSaveGridConfig()
+        {
+            var (org, cookie) = await CreatePrerequisites();
+
+            var columns = CreateTestColumns().ToList();
+            var saveResponse = await OrganizationGridConfigurationTestHelper.SendSaveConfigurationRequestAsync(org.Uuid, columns, cookie);
+            Assert.Equal(HttpStatusCode.OK, saveResponse.StatusCode);
+        }
+
+        [Fact]
+        public async Task RegularUserCanGetGridConfiguration()
+        {
+            //Admins saves a column configuration
+            var columns = CreateTestColumns().ToArray();
+            var (org, localAdminCookie) = await CreatePrerequisites();
+            _ = await OrganizationGridConfigurationTestHelper.SendSaveConfigurationRequestAsync(org.Uuid, columns, localAdminCookie);
+
+            //User fetches column configuration
+            var (_, _, userCookie) = await HttpApi.CreateUserAndLogin(CreateEmail(), OrganizationRole.User, org.Id);
+            var userResponse =
+                await OrganizationGridConfigurationTestHelper.SendGetConfigurationRequestAsync(org.Uuid, userCookie);
+            var userResponseBody = await userResponse.ReadResponseBodyAsAsync<OrganizationGridConfigurationResponseDTO>();
+
+            Assert.Equal(HttpStatusCode.OK, userResponse.StatusCode);
+            foreach (var column in userResponseBody.VisibleColumns)
+            {
+                Assert.Contains(column.PersistId, columns.Select(x => x.PersistId));
+            }
+        }
+
+        [Fact]
+        public async Task LocalAdminCanDeleteGridConfiguration()
+        {
+            var columns = CreateTestColumns().ToArray();
+            var (org, localAdminCookie) = await CreatePrerequisites();
+            _ = await OrganizationGridConfigurationTestHelper.SendSaveConfigurationRequestAsync(org.Uuid, columns, localAdminCookie);
+
+            var deleteResponse = await 
```

Comment:
add a "using" -> using var deleteResponse

So it later on dissolves it

Reply:
If you can also add it to Tests.Integration.Presentation.Web/Tools/External/OrganizationUnitV2Helper.cs line 57 (I also forgot to use it)